.github/workflows/docs.yaml

```diff
@@ -47,4 +47,4 @@ jobs:
 
     - name: Check the API documentation
       run: |
-        tox -q -e apidocs
+        tox --quiet --skip-pkg-install -e apidocs
```

Comment:
I think that with --skip-pkg-install we can enable back non-quiet mode.

The -q was there to suppress the deps installation info